openslides/users/static/templates/users/user-list.html

```diff
@@ -1,10 +1,136 @@
+      <i class="fa fa-file-pdf-o fa-lg"></i>
+      <translate>PDF
+    </a>
+    <button os-perms="users.can_manage" class="btn btn-default btn-sm dropdown-toggle"
+        data-toggle="dropdown">
+      <i class="fa fa-caret-down"></i>
+    </button>
+    <ul class="dropdown-menu dropdown-menu-right">
+      <li><a href="/users/print/" target="_blank">
+            <i class="fa fa-list fa-fw"></i>
+            <translate>List of participants
+          </a>
+      <li os-perms="users.can_manage">
+          <a href="/users/passwords/print/" target="_blank">
+            <i class="fa fa-qrcode fa-fw"></i>
+            <translate>List of access data
+          </a>
+    </ul>
+  </div>
+</div>
+
+<div class="row form-group">
+  <div class="col-sm-8">
+    <input type="checkbox" ng-model="filterPresent">
+    <translate>Is present
+  </div>
+  <div class="col-sm-4">
+    <input type="text" os-focus-me ng-model="filter.search" class="form-control"
+        placeholder="{{'Filter'|translate}}">
+  </div>
+</div>
+
+<table class="table table-striped table-bordered table-hover">
+  <thead>
+    <tr>
+      <th>
+        <span ng-click="sortby='is_present';reverse=!reverse">
+          <translate>Present</translate>
+          <i class="fa" ng-show="sortby == 'is_present'" 
+              ng-class="reverse ? 'fa-sort-desc' : 'fa-sort-asc'"></i>
+        </span>
+      <th>
+        <span ng-click="sortby='first_name';reverse=!reverse">
+          <translate>Name</translate>
+          <i class="fa" ng-show="sortby == 'first_name'"
+              ng-class="reverse ? 'fa-caret-down' : 'fa-caret-up'"></i>
+        </span>
+      <th class="optional">
+        <span ng-click="sortby='structure_level';reverse=!reverse">
+          <translate>Structure level</translate>
+          <i class="fa" ng-show="sortby == 'structure_level'"
+              ng-class="reverse ? 'fa-caret-down' : 'fa-caret-up'"></i>
+        </span>
+      <th class="optional">
+        <span ng-click="sortby='groups';reverse=!reverse">
+          <translate>Group</translate>
+          <i class="fa" ng-show="sortby == 'groups'"
+              ng-class="reverse ? 'fa-caret-down' : 'fa-caret-up'"></i>
+        </span>
+      <th os-perm="users.can_manage" class="optional">
+        <span ng-click="sortby='comment';reverse=!reverse">
+          <translate>Comment</translate>
+          <i class="fa" ng-show="sortby == 'comment'"
+              ng-class="reverse ? 'fa-caret-down' : 'fa-caret-up'"></i>
+        </span>
+      <th os-perm="users.can_manage" class="optional">
+        <span ng-click="sortby='last_login';reverse=!reverse">
+          <translate>Last Login</translate>
+          <i class="fa" ng-show="sortby == 'last_login'"
+              ng-class="reverse ? 'fa-caret-down' : 'fa-caret-up'"></i>
+        </span>
+      <th os-perm="users.can_manage core.can_manage_projector" class="mini_width">Actions
+    </tr>
+  </thead>
+  <tbody>
```

Comment:
You can decide if you want to remove the closing thead-element or the starting tbody element:
```A tbody element's start tag may be omitted if the first thing inside the tbody element is a tr element, and if the element is not immediately preceded by a tbody, thead, or tfoot element whose end tag has been omitted. (It can't be omitted if the element is empty.)```